Automattic/jetpack
# My Jetpack: add package version to main class and initial state

Add the My Jetpack package version to code

#### Changes proposed in this Pull Request:

* Add the My Jetpack package version to code

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check the code
* Check the version matches the latest tag https://github.com/Automattic/jetpack-my-jetpack/tags
* Visit My Jetpack and verify the value is available in `myJetpackInitialState.myJetpackVersion`
